service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/FeeThresholds.kt

```diff
@@ -5,62 +5,12 @@
 package fi.espoo.evaka.invoicing.domain
 
 import fi.espoo.evaka.shared.domain.DateRange
+import org.jdbi.v3.core.mapper.Nested
 import java.math.BigDecimal
 import java.util.UUID
 
-data class FeeThresholdsWithValidity(
```

Comment:
The idea is just to drop the version with `id` since the field is not used anywhere else apart from the editor. It's just a straightforward refactor, there's nothing domain specific really.